src/backend/common/deferred/handlers/defer_handler.py

```diff
@@ -32,8 +31,7 @@ def run(data: bytes) -> Any:
 
 
 def handle_defer(path: str) -> Response:
-    # Ignore XSRF protection in dev env
-    if not Environment.is_dev() and "X-AppEngine-TaskName" not in request.headers:
+    if "X-AppEngine-TaskName" not in request.headers and "X-Google-TBA-RedisTask" not in request.headers:
```

Comment:
- [ ] We might wanna hard-code these or something. We do use these in a few different places. Maybe we make a `consts` file for these magic header constants?